pkg/buildscript/queries.go

```diff
@@ -147,7 +150,23 @@ func getVersionRequirements(v *Value) []types.VersionRequirement {
 	return reqs
 }
 
-func (b *BuildScript) getSolveNode() (*Value, error) {
+func isSolveFuncName(name string) bool {
+	return name == solveFuncName || name == solveLegacyFuncName
+}
+
+func (b *BuildScript) getTargetNode(target string) (*Value, error) {
```

Comment:
nit: Any taste for a variadic argument here? Would allow us to forgo the passing of `""` and be ready for multiple targets when the time comes.

Reply:
Yes! I was looking for an excuse to make this an optional argument, and multiple target support justifies it!